daily-report-dashboard/flask-frontend/templates/proxy_filings_summary.html

```
@@ -0,0 +1,94 @@
+{% extends "main.html" %}
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)